tuf/data/types.go

```diff
@@ -111,6 +111,7 @@ type Signature struct {
 	KeyID     string       `json:"keyid"`
 	Method    SigAlgorithm `json:"method"`
 	Signature []byte       `json:"sig"`
+	IsValid   bool         `json:"-isvalid"`
```

Comment:
I think `json:"-"` is the syntax for skipping importing/exporting the field?  It might be also be good, just for our own sanity since `VerifySignatures` is now setting this value, to add a test that marshals a `data.Signed` object using canonical JSON before calling `VerifySignatures`, and then marsahalling it again after and ensuring that the bytes haven't changed.  This will make sure the `IsValid` bool is not being exported.